bundles/org.openhab.binding.livisismarthome/src/main/java/org/openhab/binding/livisismarthome/internal/handler/LivisiBridgeHandler.java

```diff
@@ -192,6 +199,9 @@ private void startClient() {
                 scheduleBridgeRefreshJob(bridgeDevice);
 
                 startWebSocket(bridgeDevice);
+
+                // Update the restart channel to OFF to enable a (new) restart command.
+                updateState(CHANNEL_RESTART, OnOffType.OFF);
```

Comment:
If you set [auto-update policy](https://next.openhab.org/docs/developer/bindings/thing-xml.html#auto-update-policies) to **veto**, there is no need to reset the state.

Reply:
Hm, will such rules still work when "veto" is used? "Item <itemName> changed to ON"? Maybe someone would like to trigger more actions when the restart is triggered.

Reply:
Yes, I would think so. `veto` is used for example by the Hunter Douglas PowerView binding to trigger scenes. This prevents the state from changing:
```
2024-07-03 21:03:57.550 [INFO ] [openhab.event.ItemCommandEvent      ] - Item 'PowerViewHub_Scene_Office_Up' received command ON
```

So you should be able to use (see https://www.openhab.org/docs/configuration/rules-dsl.html#event-based-triggers):
```java
Item <item> received command [<command>]
```

You could give it a try. 😉

Reply:
No, unfortunately it doesn't work. This rule is working with my committed version, but when I add <autoUpdatePolicy>veto</autoUpdatePolicy> the rule isn't working anymore. Actually I would write the rule with "changed to ON" to let the rule only react in one direction, this is also only working with the committed version, not with veto. Therefore I would like to leave the code as it is.

rule "Restart Rule"
when
    Item LIVISI_SmartHome_Zentrale_Restart_2 changed
then
    RestartRuleTriggered.sendCommand(ON)
end

Reply:
So this did not work?

```java
rule "Restart Rule"
when
Item LIVISI_SmartHome_Zentrale_Restart_2 received command ON
then
RestartRuleTriggered.sendCommand(ON)
end
```

If indeed this is not working, that's strange, and I would need to look into that.

Reply:
Ok, it is working with "received command ON". I have now changed the channel to veto and committed and pushed that change. But I thing it has a very bad usability. How should the (end-)user know that it the channel is configured with veto and that it means that a rule has to be written with "received command" instead of "changed"? This detail is also not displayed within Main UI. I have updated the README to hint to veto, but I think you should at least extend the OpenHAB documentation to explain what veto means and how to write rules for such channels.

Reply:
I don't think you should use the terms "auto-update policy veto" in the README since these are technical developer-oriented terms. How would the user know? Simply by looking at the event.log like mentioned [here](https://github.com/openhab/openhab-addons/pull/16969#discussion_r1664631547).

I'm not sure it's a common use-case to have a rule trigger on this event which is already triggered by the user or a rule in the first place, in order to restart the controller? But if you want to be really verbose, you could provide a rule example in the example chapter (i.e. close to "Rules example for push-buttons"), and link to that from the channel description, WDYT?

Reply:
That's the problem. The user needs to know technical details to understand the different behaviour of this switch channel, at least when writing a rule. With this hint in the readme the user can at least google for it to find the OpenHAB documentation (which should get enhanced to.describe the behaviour in rules). You were also not sure how the behaviour is in rules... ;-) And no, the user shouldn't have to look in the events.log file before writing a rule for a switch item... 

Maybe you could also support the change triggers for veto. That should be backward compatible because such rules are currently not working (therefore there can't exist working rules which would break with that fix).

I could remove the hint within the readme and add a comment within the channel.xml of the binding. That would help me and other developers to still know the different behaviour when issues come up. Would that be better to get an approve of the pull-request?

Reply:
Did you see this proposal?
>> you could provide a rule example in the example chapter (i.e. close to "Rules example for push-buttons"), and link to that from the channel description, WDYT?

To answer the rest:
> Maybe you could also support the change triggers for veto.

I'm not sure what you mean by this?

> I could remove the hint within the readme and add a comment within the channel.xml of the binding.

I'm only opposing using the terms "auto-update policy" and "veto" in the user documentation, because those are developer concept. The channel receives commands from the binding, but it doesn't receive state updates from the binding. This is important for the user to know.

The means to achieve this behavior, for you as a developer, is to use the `veto` policy. However, the "auto-update" concept is the same as described here:
https://www.openhab.org/docs/configuration/items.html#parameter-autoupdate

So if the binding didn't refrain from updating the state, the user could override this by setting the `autoupdate` parameter to `false`.

Reply:
I also found this post with useful insights on this topic: https://community.openhab.org/t/autoupdate-a-primer/101509

Reply:
Gentle ping @Novanic does this provide enough details to proceed?

Reply:
I have now removed the auto-update veto hint from the README. I don't have documented an example rule for this channel, because I would rather have to document that "changed" and "received update" isn't working for this special channel (and the most users wouldn't write a rule for that channel, therefore the documentation would be confusing for the most users).